content/zh-cn/docs/contribute/style/style-guide.md

```diff
@@ -391,6 +381,79 @@ Set the value of the `replicas` field to 2. | Set the value of the `replicas` fi
 将 `replicas` 字段值设置为 2. | 将 `replicas` 字段值设置为 `2`.
 {{< /table >}}
 
+<!--
+## Referring to Kubernetes API resources
+
+This section talks about how we reference API resources in the documentation.
+
+### Clarification about "resource"
+
+Kubernetes uses the word "resource" to refer to API resources, such as `pod`, `deployment`, and so on. We also use "resource" to talk about CPU and memory requests and limits. Always refer to API resources as "API resources" to avoid confusion with CPU and memory resources.
+-->
+## 引用 Kubernetes API 资源
+
+本节讨论我们如何在文档中引用 API 资源。
+
+### 澄清“资源”
+
+Kubernetes 使用“resource”一词来指代 API 资源，例如 `pod`、`deployment` 等。 
```

Comment:
```suggestion
Kubernetes 使用 “resource” 一词来指代 API 资源，例如 `pod`、`deployment` 等。 
```
If possible, give a space around the quotation mark `“”`.

Reply:
Thanks, done. I have added space around all the quotation mark `“”`.